spikeinterface/toolkit/postprocessing/correlograms.py

```diff
@@ -92,3 +232,56 @@ def compute_correlograms(sorting,
         bins = np.arange(correlograms.shape[2] + 1) * real_bin_duration_ms
 
     return correlograms, bins
+
+
+def compute_correlograms_numba(sorting,
+                         window_ms=100.0, bin_ms=5.0,
+                         symmetrize=False):
+    """
+    Computes several cross-correlogram in one course
+    from several cluster.
+    
+    This is a "brute force" method using compiled code (numba)
+    to accelerate the computation.
+    
+    Adaptation: Aurélien Wyngaard
+    """
+
+    fs = sorting.get_sampling_frequency()
+    num_units = len(sorting.unit_ids)
+
+    window_size = int(fs * window_ms/2 * 1e-3)
+    bin_size = int(fs * bin_ms * 1e-3)
+    real_bin_duration_ms = bin_size / fs * 1e3
+    num_bins = 2 * int(window_size / bin_size)
+
+    bins = np.arange(-window_size, window_size+bin_size, bin_size) * 1e3 / fs
+    spikes = sorting.get_all_spike_trains(outputs='unit_index')
+
+    assert num_bins >= 1
+
+    correlograms = np.zeros((num_units, num_units, num_bins), dtype=np.int64)
+
+    for seg_index in range(sorting.get_num_segments()):
+        _compute_correlograms_numba(correlograms, spikes[seg_index][0].astype(np.int64),
+                                    spikes[seg_index][1].astype(np.int32),
+                                    window_size, bin_size, fs)
+    
+    return correlograms, bins
+
+@numba.jit((numba.int64[:, :, ::1], numba.int64[::1], numba.int32[::1], numba.int32, numba.int32, numba.float32),
+            nopython=True, nogil=True, cache=True, parallel=True)
+def _compute_correlograms_numba(correlograms, spike_trains, spike_clusters, max_time, bin_size, sampling_f):
```

Comment:
You should put this in a blockwith
`if HAVE_NUMBA:`

Reply:
Done.